stan/math/prim/prob/exp_mod_normal_lccdf.hpp

```diff
@@ -48,69 +48,65 @@ return_type_t<T_y, T_loc, T_scale, T_inv_scale> exp_mod_normal_lccdf(
   scalar_seq_view<T_inv_scale> lambda_vec(lambda);
   size_t N = max_size(y, mu, sigma, lambda);
 
-  for (size_t n = 0; n < N; n++) {
+  for (size_t n = 0, size_y = size(y); n < size_y; n++) {
     if (is_inf(y_vec[n])) {
-      if (y_vec[n] > 0.0) {
-        return ops_partials.build(negative_infinity());
-      } else {
-        return ops_partials.build(0.0);
-      }
+      return y_vec[n] > 0.0 ? ops_partials.build(negative_infinity())
```

Comment:
This is all optional as I'm not a reviewer, but I prefer the conditional operator to be pushed inside to remove redundant code, e.g.,

```
return ops_partials.build(y_vec[n] > 0 ? negative_infinity() : 0);
```

Also, it's not necessary to use `0.0` instead of `0` anywhere but in integer division contexts.

Reply:
Nice one, I'm glad you mentioned!